official/recommendation/data_preprocessing.py

```diff
@@ -494,10 +502,6 @@ def make_train_input_fn(ncf_dataset):
   # type: (NCFDataset) -> (typing.Callable, str, int)
   """Construct training input_fn for the current epoch."""
 
-  if not tf.gfile.Exists(ncf_dataset.cache_paths.subproc_alive):
```

Comment:
Can you keep this in, but just change the error message. The subprocess can still die, so it should still check.